app/assets/stylesheets/components/_contact-record.scss

```diff
@@ -19,6 +19,12 @@
       padding-right: 0.75rem;
       background-color: white;
     }
+    .contact-record__subject::before {
+      content: "✉️ ";
+    }
```

Comment:
If this is intended as semantic content to indicate that this is an email, I'd put it in the html. If it's purely a visual accent and does not add significant information, it's fine to keep it in a pseudoelement.